test/new-e2e/tests/npm/ec2_1host_selinux_test.go

```diff
@@ -0,0 +1,138 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package npm
+
+import (
+	"testing"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/e2e"
+	awshost "github.com/DataDog/datadog-agent/test/new-e2e/pkg/environments/aws/host"
+
+	compos "github.com/DataDog/test-infra-definitions/components/os"
+	"github.com/DataDog/test-infra-definitions/scenarios/aws/ec2"
+)
+
+type ec2VMSELinuxSuite struct {
+	e2e.BaseSuite[hostHttpbinEnv]
+}
+
+// TestEC2VMSuite will validate running the agent on a single EC2 VM
+func TestEC2VMSELinuxSuite(t *testing.T) {
+	s := &ec2VMSELinuxSuite{}
+
+	e2eParams := []e2e.SuiteOption{e2e.WithProvisioner(
+		e2e.NewTypedPulumiProvisioner("hostHttpbin", hostDockerHttpbinEnvProvisioner(awshost.WithEC2InstanceOptions(
+			// RHEL9
+			ec2.WithAMI("ami-0fe630eb857a6ec83", compos.AmazonLinux2, compos.AMD64Arch), ec2.WithInstanceType("t3.medium"))), nil)),
+	}
+
+	// Source of our kitchen CI images test/kitchen/platforms.json
+	// Other VM image can be used, our kitchen CI images test/kitchen/platforms.json
+	// ec2params.WithImageName("ami-a4dc46db", os.AMD64Arch, ec2os.AmazonLinuxOS) // ubuntu-16-04-4.4
+	e2e.Run(t, s, e2eParams...)
+}
+
+// BeforeTest will be called before each test
+func (v *ec2VMSELinuxSuite) BeforeTest(suiteName, testName string) {
+	v.BaseSuite.BeforeTest(suiteName, testName)
+
+	// default is to reset the current state of the fakeintake aggregators
+	if !v.BaseSuite.IsDevMode() {
+		v.Env().FakeIntake.Client().FlushServerAndResetAggregators()
+	}
+}
+
+func (v *ec2VMSELinuxSuite) SetupSuite() {
+	v.BaseSuite.SetupSuite()
+
+	v.Env().RemoteHost.MustExecute("sudo yum install -y bind-utils httpd-tools")
+	v.Env().RemoteHost.MustExecute("sudo yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo")
+	v.Env().RemoteHost.MustExecute("sudo yum install -y docker-ce docker-ce-cli")
+	v.Env().RemoteHost.MustExecute("sudo systemctl start docker")
+	v.Env().RemoteHost.MustExecute("sudo usermod -a -G docker $(whoami)")
+	v.Env().RemoteHost.ReconnectSSH()
+
+	// prefetch docker image locally
+	v.Env().RemoteHost.MustExecute("docker pull public.ecr.aws/k8m1l3p1/alpine/curler:latest")
+	v.Env().RemoteHost.MustExecute("docker pull public.ecr.aws/docker/library/httpd:latest")
+	v.Env().RemoteHost.MustExecute("docker pull public.ecr.aws/patrickc/troubleshoot-util:latest")
```

Comment:
These docker are here just to run a executable inside a container like customer do.
In case of 1host tests, docker manager is not installed and would require a special option just for that.
I would prefer to keep the testing network traffic outside of pulumi as it's not dependent of the infra
WDYT ?